samples/README.md

```diff
@@ -0,0 +1,9 @@
+Redwood Samples
+===============
+
+This project requires locally publishing the enclosing redwood project before its own Gradle build
+can be launched. These artifacts must also be re-published each time the enclosing project changes.
+
+```
+./gradlew publishAllPublicationsToLocalMavenRepository -PVERSION_NAME=1.0-local
```

Comment:
I couldn’t find a satisfying way to make Gradle run this automatically for me, in a way that the Redwood plugin was available to the build script itself.

If you have ideas, running this manually suuuucks. Particularly because if you forget to run it your samples run on stale Redwood code.